release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3736,4 +3736,30 @@ $ oc adm release info 4.12.20 --pullspecs
 [id="ocp-4-12-20-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-21"]
+=== RHBA-2023:3546 - {product-title} 4.12.21 bug fix update and security update
+
+Issued: 2023-06-14
+
+{product-title} release 4.12.21, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3546[RHBA-2023:3546] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3545[RHSA-2023:3545] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.21 --pullspecs
+----
+
+[id="ocp-4-12-21-bug-fixes"]
```

Comment:
The fix made it into OCP as part of 4.12.20 z stream release. Is it possible to get that z stream version documentation updated as well?

Reply:
It is possible but it would need to go through the Change Management process.

Reply:
Can you please elaborate on that? Is there anything needed from my side to enable that process?

Reply:
It depends a bit on exactly what you're wanting to happen. Do you want to remove the bug from 4.12.21 and move it to 4.12.20, or just add it to 4.12.20 in addition to 4.12.21?